jfix-armeria-aggregating-profiler/src/test/kotlin/ru/fix/armeria/aggregating/profiler/ProfilerTestUtils.kt

```diff
@@ -0,0 +1,18 @@
+package ru.fix.armeria.aggregating.profiler
+
+import com.linecorp.armeria.common.SessionProtocol
+import com.linecorp.armeria.testing.junit.server.ServerExtension
+import java.net.URI
+
+object ProfilerTestUtils {
+
+    const val LOCAL_HOST = "localhost"
+    const val LOCAL_ADDRESS = "127.0.0.1"
+    const val EPOLL_SOCKET_CHANNEL = "EpollSocketChannel"
```

Comment:
maybe me can eval it from class name
```suggestion
    val EPOLL_SOCKET_CHANNEL = EpollSocketChannel::class.simpleName!!
```

Reply:
the problem here is that lib hosting EpollSocketChannel is a runtime dependency

![image](https://user-images.githubusercontent.com/9992425/87400989-3a42b900-c5c2-11ea-9854-8a125281d7af.png)

Do you think it is better to declare it as test compile dependency?

Reply:
ok, let it be just a string constant.